src/org_parser/transform.cljc

```diff
@@ -36,7 +43,20 @@
   (->> props
        (filter #(= prop (first %)))
        first
-       (drop 1)))
+       (drop 1)
+       vec))
+
+(defn- extract-tags [[_ s]]
+  "Given a [:text-normal 'xxx'], return the text-normal without tags
+  and the tags as a list."
+  (let [[tags & _] (re-find #"\s+(:[a-zA-Z0-9_@#%]+)+:\s*$" s)] ;; find tags by regex
+    (if (nil? tags)
+      [[:text-normal s] []]
+      [[:text-normal (subs s (- (count s) (count tags)))]
+       (->> tags str/trim #(str/split % #":") (filter #(not (= % ""))))])))
```

Comment:
@munen in the next two lines I explain what this should do.

The tags must be extracted here, not in EBNF for practical reasons, see EBNF.